src/execution/values.js

```diff
@@ -165,6 +165,31 @@ export function getArgumentValues(
 }
 
 /**
+ * Prepares an object map of directive name to argument values given a list of
+ * directive definitions and AST node.
+ */
+export function getAppliedDirectives(
```

Comment:
Or - what do you think about having this be a singular function - `getAppliedDirective()` which accepts a `GraphQLDirective` and a node and returns `void | { [string]: mixed }`?